src/builtins/compiled/now.rs

```diff
@@ -47,6 +47,6 @@ impl Now {
         let timezone = timezone.unwrap_or(TimeZone::IanaIdentifier(sys::get_system_timezone()?));
         let system_nanos = sys::get_system_nanoseconds()?;
         let epoch_nanos = EpochNanoseconds::try_from(system_nanos)?;
-        Now::plain_time_iso_with_provider(epoch_nanos, timezone, &*provider)
+        Now::plain_time_iso_with_provider_and_system_info(epoch_nanos, timezone, &*provider)
```

Comment:
Thought: I'm not against merging this as it is, but I think it would be worth looking into having a builder API instead of two functions with slightly different arguments. Could be hard the same semantics through FFI though, so it'd require a bit of thinking.

Reply:
Yeah, I'm up for adjustments or rethinking `Now`'s API. A builder pattern could actually be really nice. The main goal here was to mostly align the method names a little bit better with the rest of the crate and add tests.

I'm really not in love with the names, but at the very least they are specific. We should open an issue for the builder pattern.